pkg/backup/job.go

```diff
@@ -78,7 +78,10 @@ func (b *BackupJob) Run() {
 		return
 	}
 
-	if err := b.storage.Copy(localfile, filepath.Base(localfile)); err != nil {
+	remotefile := filepath.Join(b.conf.name,
```

Comment:
Should the file have a prefix or do you want to create a new folder? As far as I know `filepath.Join("a","b","c")` would return `/a/b/c`.